### PR TITLE
[CLI] Add CLI e2e for move compile command

### PR DESCRIPTION
**Pull request by 0xjinn**
### Test Plan

```
2023-06-22 16:28:56,301 - INFO - These tests passed:
2023-06-22 16:28:56,301 - INFO - test_metrics_accessible
2023-06-22 16:28:56,301 - INFO - test_init
2023-06-22 16:28:56,301 - INFO - test_account_fund_with_faucet
2023-06-22 16:28:56,301 - INFO - test_account_create
2023-06-22 16:28:56,302 - INFO - test_account_lookup_address
2023-06-22 16:28:56,302 - INFO - test_aptos_header_included
2023-06-22 16:28:56,302 - INFO - test_move_compile
2023-06-22 16:28:56,302 - INFO - test_move_compile_script
2023-06-22 16:28:56,302 - INFO - test_move_publish
2023-06-22 16:28:56,302 - INFO - All tests passed!
```

### CODE REVIEW

File: crates/aptos/e2e/main.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def run_tests(run_helper):
 
     # Run move subcommand group tests.
     test_move_publish(run_helper)
+    test_move_compile(run_helper)
```
Comment (banool) [inline crates/aptos/e2e/main.py:120]: Nit: Put this before publish, compile is just a subset of publish, it'd be nice to see first that compile succeeds if publish fails. It doesn't actually matter, it's just a brain thing hahah.

**Review by banool — APPROVED** [review]
Clean!!!!

File: crates/aptos/e2e/cases/move.py [hunk 2]
```diff
@@ -59,3 +59,26 @@ def test_move_publish(run_helper: RunHelper, test_name=None):
...
+
+
+@test_case
+def test_move_compile(run_helper: RunHelper, test_name=None):
+    package_dir = f"move/{run_helper.base_network}"
+    account_info = run_helper.get_account_info()
+
+    # Compile the module.
+    response = run_helper.run_command(
+        test_name,
```
Comment (0xmaayan) [inline crates/aptos/e2e/cases/move.py:71]: why do we do `test_name` as `None`..?

Comment (0xmaayan) [reply crates/aptos/e2e/cases/move.py:71]: noice

**Review by 0xmaayan — APPROVED** [review]

**Review by gregnazario — APPROVED** [review]

**Comment by 0xjinn** [comment]
Update: Added a new script file for testing `aptos move compile-script`

**Review by banool — APPROVED** [review]
Awesome, much cleaner this way!

**Pull request merged**